pkg/postage/postagecontract/contract.go

```diff
@@ -118,7 +118,7 @@ func (c *postageContract) sendTransaction(ctx context.Context, callData []byte,
 		To:          &c.postageContractAddress,
 		Data:        callData,
 		GasPrice:    sctx.GetGasPrice(ctx),
-		GasLimit:    1600000,
+		GasLimit:    30000000,
```

Comment:
not a good idea. this tx can only be included if it is the only tx in the block (also you need to be able to afford the entire transaction even if it is only a lot less).